src/ui/public/visualize/loader/embedded_visualize_handler.js

```diff
@@ -17,7 +17,11 @@
  * under the License.
  */
 
+import _ from 'lodash';
```

Comment:
I think we should rather start using named imports for that (since it would potentially allow for better tree shaking - and imho also improves readability): `import { debounce } from 'lodash';`